app/api/search/route.ts

```
@@ -0,0 +1,92 @@
+  if (error || !data) return null;
+  for (const row of data) {
+    if (!row.uri) continue;
+    try {
+      const metadata = await fetchTokenMetadata(
+        row.address,
+        String(row.tokenId)
+      );
+      if (
+        metadata &&
+        metadata.name &&
+        metadata.name.toLowerCase().startsWith(searchKey.toLowerCase())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)